site/content/docs/monitoring/check-pricing.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Check Pricing
+weight: 9
+menu:
+   platform:
+    parent: "Monitoring"
+---
+
+Check pricing is based on a number of factors. Here is a breakdown of the things that determine the cost of a check.
+
+### Base cost
+
+Browser checks and API checks have a base price per check run. Always refer to our [pricing page](https://www.checklyhq.com/pricing/) for the latest prices.
+
+The base cost is typically listed as $X per 1k or 10k check runs.
+
+If a browser check is running once per hour with a cost of $5 per 1000 check runs, the monthly base cost for the check would be $3,65.
+
+### Parallel scheduling
+
+When a check is running in [parallel](/docs/monitoring/global-locations/#parallel), it will run once on each selected location per execution. If you change a check from using the round-robin scheduling method to running in parallel, be aware that the cost will multiply by the number of locations you have selected.
+
+Balance the number of locations the check is using to ensure you are quickly made aware of potential issues in critical locations without running an unnecessary amount of checks.
+
+Continuing our example from earlier, if we want our Browser check to run in parallel from 5 locations, the monthly cost will increase from $3,65 to $18,25.
+
+### Retries
+
+When a check is [retried](/docs/alerting-and-retries/retries), this counts as a new check run. A flaky check can increase your check run costs above the expected. If you have problems with flaky checks, ask our community or our support for tips on how to improve check stability and reduce cost.
+
+If our check from earlier has a 20% retry rate, this will increase our cost from $3,65 to $4,40.
```

Comment:
I would change this from "our cost" to "the cost" or "your cost" to make it as unambiguous as possible